### PR TITLE
Remove extrenuous log statements from persistent timestamp service

### PR DESCRIPTION
**Pull request by gbrova**
@carrino 

Unclear that this error is useful, since we immediately recover by allocating more timestamps. 

This PR removes the log altogether.  Alternatively, we could change it to something like `log.info("Not enough timestamps were preallocated, so we requested a new batch")`

### CODE REVIEW

**Comment by joelea** [comment]
@clockfort is it reasonable to ad the logback dependency here? I don't know what the resolution of the other issue was? This could easily be changed to log4j, we were just using logback elsewhere. It's also a test dependency so it won't matter in the near future.

**Comment by jboreiko** [comment]
I would rather see this moved to info or debug, seems like a major step in the timestamp services function.  I would also like to see a ticket addressing the choice between logback and log4j.  From there we should standardize on one or the other.

**Comment by gbrova** [comment]
@jboreiko I think the interesting step is inside `submitAllocationTask()`, not here -- but as I said at the top I'm happy to include a log.info :-)

**Comment by clockfort** [comment]
@joelea I don't want logback dependency just because I don't want extra library bloat and it's use is trivial / small / not justified.   @jaapweel mentioned in the chat as well that logback and log4j on the same classpath will lead to slf4j complaining and picking the first one on the classpath. @twilson-palantir also expressed surprise it even got past the silent conflict checker we have.

I think it's best to remove the dependency.

**Comment by joelea** [comment]
Cool, I'm happy to remove the logback dependency until we get the internal dependency issue sorted out - then this dependency is irrelevant to everything not directly inside this subproject.

**Comment by gracew** [comment]
ran into this today and was initially alarmed at the stack trace, especially since it appears to be an NPE:
```
ERROR [2016-05-19 05:19:57,511] com.palantir.timestamp.PersistentTimestampService: We haven't gotten enough timestamps from the DB
! java.lang.RuntimeException: null
! at com.palantir.timestamp.PersistentTimestampService.getFreshTimestamps(PersistentTimestampService.java:182) [timestamp-impl-0.4.0.jar:na]
! at com.palantir.timestamp.PersistentTimestampService.getFreshTimestamp(PersistentTimestampService.java:156) [timestamp-impl-0.4.0.jar:na]
! at com.palantir.atlasdb.transaction.impl.SnapshotTransactionManager.getImmutableTimestamp(SnapshotTransactionManager.java:242) [atlasdb-impl-shared-0.4.0.jar:na]
! at com.palantir.atlasdb.transaction.impl.SerializableTransactionManager.getImmutableTimestamp(SerializableTransactionManager.java:30) [atlasdb-impl-shared-0.4.0.jar:na]
! at com.palantir.atlasdb.transaction.impl.SnapshotTransactionManager.getApproximateImmutableTimestamp(SnapshotTransactionManager.java:237) [atlasdb-impl-shared-0.4.0.jar:na]
! at com.palantir.atlasdb.transaction.impl.SnapshotTransactionManager.runTaskReadOnly(SnapshotTransactionManager.java:191) [atlasdb-impl-shared-0.4.0.jar:na]
! at com.palantir.atlasdb.transaction.impl.SerializableTransactionManager.runTaskReadOnly(SerializableTransactionManager.java:30) [atlasdb-impl-shared-0.4.0.jar:na]
```

so +1 to changing the log level and getting rid of the randomly constructed RuntimeException.

**Comment by gbrova** [comment]
@joelea changed to log4j

**Comment by joelea** [comment]
I've just removed it until we fix the binary dependency issue - then we can add stuff like this freely.

File: timestamp-impl/src/test/resources/log4j.properties [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Root logger
```
Comment (joelea) [inline timestamp-impl/src/test/resources/log4j.properties]: Sorry, this file now needs deleting.

**Pull request merged**